VTUL/dlp-access
# fix and reorg admin page test

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2368) (:star:)

# What does this Pull Request do? (:star:)
- Fix the issue that when run tests all at once locally,  tests with user signin and singout are failed
- Rename siteAdmin_contact_config.spec.js, siteAdmin_title_config.spec.js to admin_page_contact_config.spec.js, admin_page_title_config.spec.js to make admin page tests run earlier
- Fix one security issue 

# How should this be tested?
1. Pull PR and run cypress locally
2. See Amplify preview

# Interested parties
Tag (@VTUL/dld-dev ) interested parties

(:star:) Required fields
